src/Catalog/Monitoring/Validation/Test/ValidationTestRunData.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Threading;
+using NuGet.Packaging.Core;
+using NuGet.Services.Metadata.Catalog.Helpers;
+using Microsoft.Extensions.Logging;
+
+namespace NuGet.Services.Metadata.Catalog.Monitoring.Validation.Test
+{
+    public class ValidationTestRunData
```

Comment:
what is the purpose of this class? Seems like it contains properties that belong as method parameters, like CancellationToken